Script/navBar.js

```diff
@@ -5,6 +5,15 @@ let navBarActive = document.getElementById("navBarActive").innerHTML;
 : (navBarActive == "registerPage" || navBarActive == "loginPage" || navBarActive == "adminPage") ? (document.getElementById("accountNav").classList.add("active"))
 : null;
 
+document.onkeypress = function (e) 
+{
+    e = e || window.event;
+
+    if (e.keyCode === 13)
+    {
+        searchBar();
+    }
+}
```

Comment:
Use JQuery, sent you a link on Discord, do let me know if you need any help

Reply:
This should be located in the function, otherwise whenever you press enter, it will always do the check for ticketID